python/bolt12.py

```diff
@@ -111,24 +117,174 @@ def simple_bech32_decode(bech32str: str) -> Tuple[str, bytes]:
         return None, None
```

Comment:
if sep == -1, it should throw Error(Separator not present) instead of 'return None, None'?

Reply:
Yes, it would be nice to give the user more information, but Python exceptions are an awkward way to do it unless you create a new class (imagine a GUI app, which wanted to highlight where the error was).

Hmm, maybe you're right...